scripts/wallet.js

```diff
@@ -252,12 +281,94 @@ export class Wallet {
     async getKeyToExport() {
         return await this.#masterKey?.getKeyToExport(this.#nAccount);
     }
+
+    //Get path from a script
+    async getPath(script) {
+        const dataBytes = hexToBytes(script);
+        if (isP2PKH(dataBytes)) {
+            const address = this.updatePkhMap(
+                bytesToHex(
+                    dataBytes.slice(P2PK_START_INDEX, P2PK_START_INDEX + 20)
+                )
+            );
+            return await this.isOwnAddress(address);
+        } else if (isP2CS(dataBytes)) {
+            const address = this.updatePkhMap(
+                bytesToHex(
+                    dataBytes.slice(COLD_START_INDEX, COLD_START_INDEX + 20)
+                )
+            );
+            return await this.isOwnAddress(address);
```

Comment:
Put the `if` inside the .slice (perhaps as a ternary operator) to remove the copy paste